src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -186,9 +191,8 @@ func (face *folderyMcFolderFace) setFolder(
 	if zombey, tombstoned := face.tombstones[id]; tombstoned {
 		delete(face.tombstones, id)
 
-		zombey.parent = parent
-		zombey.name = name
-		parent.children[id] = zombey
+		zombey.parent = parentNode
```

Comment:
should this also be updating the contained folder for the node like is done with the root above?

Reply:
good catch, yes.